src/core/StelUtils.cpp

```diff
@@ -2788,6 +2790,16 @@ QByteArray uncompress(QIODevice& device, qint64 maxBytes)
 	return out;
 }
 
+qint64 getLongLong(const QJsonValue& v)
+{
+	if(v.isString()) return v.toString().toLongLong();
+	const auto value = v.toDouble();
+	constexpr qint64 max = (1LL<<std::numeric_limits<double>::digits) - 1;
+	if(std::abs(value) > max) return 0;
+	const auto integer = static_cast<qint64>(value);
+	if(value != integer) return 0; // fractional part must be zero
```

Comment:
Please emit a user-friendly warning here in case of problems, else we may have strange zeroes that go unnoticed: 
qWarning() << "Cannot convert integer from JSON value "  << v << ". Please format as string value." 

If all is OK, we should never see the warning.